examples/redirect/src/main.rs

```diff
@@ -1,19 +1,9 @@
 #![feature(plugin)]
 #![plugin(rocket_codegen)]
-extern crate rocket;
-
-use rocket::response::Redirect;
-
-#[get("/")]
-fn root() -> Redirect {
-    Redirect::to("/login")
-}
 
-#[get("/login")]
-fn login() -> &'static str {
-    "Hi! Please log in before continuing."
-}
+extern crate rocket;
+extern crate redirect;
```

Comment:
Why has this been split up into a separate crate? There's no ned for that in this example. Simple add a new file `tests.rs`, and add `#[cfg(test)] mod tests` to `src/main.rs`. All of the examples, except for `config`, which is a special case, do this.